src/views/settings/Org/OrganizationList.vue

```diff
@@ -45,13 +45,22 @@ export default {
           app: 'orgs',
           resource: 'organization'
         },
-        columns: ['name',
+        columns: ['name', 'id',
           'resource_statistics.users_amount',
           'resource_statistics.groups_amount',
           'resource_statistics.assets_amount',
           'resource_statistics.asset_perms_amount',
           'actions'
         ],
+        columnsShow: {
+          min: ['name', 'actions'],
+          default: [
+            'name', 'resource_statistics.users_amount',
+            'resource_statistics.groups_amount',
+            'resource_statistics.assets_amount',
+            'resource_statistics.asset_perms_amount'
+          ]
+        },
         columnsMeta: {
           'resource_statistics.users_amount': {
             label: this.$t('Users'),
```

Comment:
There is no irregularity in this piece of code related to its functionality or performance considerations; however, there might be some stylistic/linguistic improvements that can make the text more readable: "export * from './index';". Additionally, you should always keep in mind the guidelines provided by Microsoft for readability (e.g., one tab indentation per sentence).